plugins/permission-common/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@backstage/plugin-permission-common",
+  "description": "Isomorphic types and client for Backstage permissions and authorization",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
```

Comment:
Missing `module` here, should look like this: https://github.com/backstage/backstage/blob/d7016309725f2dd738441123620470fa1e847c32/plugins/kubernetes-common/package.json#L11